src/service/video-manager-impl.js

```diff
@@ -755,11 +876,64 @@ class VideoEntry {
    *
    * @private
    */
-  unDockVideo_() {
+  finishDocking_() {
     // Restore the video inline
     this.internalElement_.classList.remove(DOCK_CLASS);
     this.internalElement_.setAttribute('style', '');
+    this.dockingState_ = DockingStates.INLINE;
     this.video.showControls();
```

Comment:
Maybe I'm not understanding what `hideControls` / `showControls` do... When we start docking videos, we call `hideControls`, which in my understanding blocks the user from using the controls (equivalent of removing `controls` in an HTML5 video). If we take off this line, then the user won't be able to bring the controls back on tap since they were disabled by `hideControls` right?